cloud/deploys/staging_azure/deploy

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
```

Comment:
i think the idea is that this lives in shared and that deploy has a .sh config file that includes anything we'd need here